app/models/federal_register_client.rb

```diff
@@ -94,6 +81,27 @@ def sorn_params(result)
     }
   end
 
+  def sorn_result?(result)
+    # We researched all Federal Register search result titles
+    # https://docs.google.com/document/d/15gwih9P6ebazWCS2ekxQ5Id1rDWbktg1mFdXtHIPZ44/edit#
+
+    # find the expected sorn titles
+    # exclude any computer matching agreements
+    result.type == 'Notice' && a_sorn_title?(result.title) && not_a_cma?(result)
+  end
+
+  def a_sorn_title?(title)
+    # titles with either privacy act or system of record
+    title.match? /(privacy act|system[\ssof]*record)/i
+  end
+
+  def not_a_cma?(result)
+    # no titles or actions with the word match
+    not_in_title = result.title.downcase.exclude?('match')
+    not_in_action = result.action.downcase.exclude?('match') if result.action.present?
+    not_in_title && not_in_action
```

Comment:
The main change of this PR. This method looks for `match` in either the title or the action.